packages/vaex-core/vaex/dataframe.py

```diff
@@ -5747,6 +5751,40 @@ def export_fits(self, path, column_names=None, shuffle=False, selection=False, p
         import vaex.export
         vaex.export.export_fits(self, path, column_names, shuffle, selection, progress=progress, virtual=virtual, sort=sort, ascending=ascending)
 
+    def export_csv(self, path, virtual=True, selection=False, progress=None, batch_size=1_000_000, **kwargs):
+        """ Exports the DataFrame to a CSV file.
+
+        :param str path: Path for filename
+        :param bool virtual: If True, export virtual columns as well
+        :param bool selection: If True, export the selection
+        :param progress: Progress callback that gets a progress fraction as argument and should return True to continue,
```

Comment:
I guess this is one of the doc snippets

Reply:
I don't understand what is the issue here, can you please explain?

Reply:
like 
`        :param progress: {progress}`
We have to reuse the doc snippets not to go crazy :)

Reply:
Ah yeah, indeed. I just copied the stuff from the other `export_` methods. Can you please check that the explanation in the doc snippets and say `export_hdf5` is consistent and has not changed recently? (just in case).

Reply:
I don't understand what you are saying, but also export_hdf5 should have used that snippet.